lightning/src/ln/channelmanager.rs

```diff
@@ -7359,7 +7364,7 @@ where
 							log_trace!(logger, "Claiming HTLC with preimage {} from our monitor", preimage);
 							self.claim_funds_internal(htlc_update.source, preimage,
 								htlc_update.htlc_value_satoshis.map(|v| v * 1000), None, true,
-								false, counterparty_node_id, funding_outpoint, channel_id);
+								false, counterparty_node_id, funding_outpoint, channel_id, None);
```

Comment:
Would we consider including this field in `MonitorEvent`s in the future so it can be available here?